diff --git a/node_modules/@react-three/drei/core/Cloud.js b/node_modules/@react-three/drei/core/Cloud.js
index 0e48115..f3d2822 100644
--- a/node_modules/@react-three/drei/core/Cloud.js
+++ b/node_modules/@react-three/drei/core/Cloud.js
@@ -4,7 +4,9 @@ import { Billboard } from './Billboard.js';
 import { Plane } from './shapes.js';
 import { useTexture } from './useTexture.js';
 
-const CLOUD_URL = 'https://rawcdn.githack.com/pmndrs/drei-assets/9225a9f1fbd449d9411125c2f419b843d0308c9f/cloud.png';
+// const CLOUD_URL = 'https://rawcdn.githack.com/pmndrs/drei-assets/9225a9f1fbd449d9411125c2f419b843d0308c9f/cloud.png';
+const CLOUD_URL = require('./cloud.png');
+
 function Cloud({
   opacity = 0.5,
   speed = 0.4,
diff --git a/node_modules/@react-three/drei/core/cloud.png b/node_modules/@react-three/drei/core/cloud.png
new file mode 100644
index 0000000..81328f2
Binary files /dev/null and b/node_modules/@react-three/drei/core/cloud.png differ
